rim/views.py

```diff
@@ -1,6 +1,9 @@
 from django.shortcuts import render
-from django.views.generic import TemplateView
+from django.views.generic import ListView
+from rim.models import Equipment
 
 
-class BaseView(TemplateView):
-    template_name='rim/base.html'
+class HomeView(ListView):
+	template_name='rim/home.html'
```

Comment:
Formatting stuff:
- You need whitespace around the equals sign
- Your editor picked tabs instead of spaces again